ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-pim-sparse-mode.j2

```diff
@@ -17,25 +17,40 @@
 {%                 endfor %}
 {%             endfor %}
 {%         endif %}
-{%         if router_pim_sparse_mode.ipv4.anycast_rps is defined and router_pim_sparse_mode.ipv4.anycast_rps is not none %}
-{%         endif %}
-{%         if router_pim_sparse_mode.ipv4.anycast_rps is defined and router_pim_sparse_mode.ipv4.anycast_rps is not none %}
+{%         if router_pim_sparse_mode.ipv4.anycast_rps is arista.avd.defined %}
 
 ### IP Anycast Information
```

Comment:
Old error Please change this heading to #### IP Anycast Information

Reply:
Done